Makefile

```diff
@@ -435,9 +450,21 @@ ifeq ($(TARGET_WINDOWS),1)
 else
   LD := $(CC)
 endif
+
+ifneq ($(TARGET_OSX),1)
 CPP := cpp -P
+else
+CPP := cpp-9 -P
+endif
+
+ifneq ($(TARGET_OSX),1)
 OBJDUMP := objdump
 OBJCOPY := objcopy
+else
+OBJDUMP := i686-w64-mingw32-objdump
+OBJCOPY := i686-w64-mingw32-objcopy
```

Comment:
llvm-objdump and llvm-objcopy should be tried, if macos ships those

Reply:
@Jan200101 LLVM isn't included with the XCode command line tools, but the `llvm` package can be installed via Homebrew. After adding `/usr/local/opt/llvm/bin` to the `PATH`, they can be accessed, but compiling the PC port with them produces a binary that hangs indefinitely.
I have also tried using `gobjcopy` & gobjdump` that are included with the `binutils` package, but as with `LLVM`, that also produces a broken binary.